packages/@react-spectrum/s2/src/ProgressCircle.tsx

```diff
@@ -503,6 +501,8 @@ function ProgressCircle(props: ProgressCircleProps, ref: DOMRef<HTMLDivElement>)
             <div
               className={isIndeterminate ? fillSubMask1Indeterminate : fillSubMask}
               style={subMask1Style}>
+              {/* TODO if we want rounded fill here then we need a div acting as a psudo element in the below div? Probably easier to just
+                  make a svg with stroke-linecap but I couldn't find the animation definition for indeterminate */}
```

Comment:
Spent some time trying to add the rounded caps to the ProgressCircle fill but it proved a bit difficult. Maybe we should just make a svg for the spinner and use stroke-line cap?

Reply:
issue with that approach was svg animations didn't have wide enough support and we can't really handle it with the current CSS solution.

though I just looked up the SVG animate element, and it appears to have wide support, I don't recall the specifics for this

Reply:
Could we have three objects, current animation and circles at start and end, that are animated at the same time?

Reply:
potentially? It just feels like doing something like https://www.fffuel.co/svg-spinner/ would be easiest